### PR TITLE
feat: Add 500px

### LINKED ISSUE #263: [BRAND REQUEST] 500px
### Please enter the brand name.

500px

### Please enter the brand website.

https://500px.com/

### Please enter the brand SVG. (If Applicable)

500px

### PR DESCRIPTION
**Pull request by KieranRobson**
# Proposed Changes

- Closes #263 
- Variables are `FIVEHUNDREDPX` as `500px` didnt work

Screenshot (If Applicable)
image

## Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [x] Included a screenshot (if adding a new button)
- [x] 🚀

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,8 +3,7 @@
 version: "3.0"
 services:
   little-link:
-    image: docker.io/library/link
-    # ghcr.io/techno-tim/littlelink-server:latest
+    image: ghcr.io/techno-tim/littlelink-server:latest
```
Comment (timothystewart6) [inline docker-compose.yml:6]: This was meant to use for local debugging, any way you can revert this?

**Comment by timothystewart6** [comment]
Any way you can add these changes to the end of the files please?  Thank yoU!

**Comment by KieranRobson** [comment]
@timothystewart6 I have moved them down to the bottom!

**Comment by timothystewart6** [comment]
Looks like there's some lint, you can fix this by running `yarn lint --fix`  

Also, can you move all of your changes to the end of the file please?

**Comment by timothystewart6** [comment]
Thank you so much!!

**Pull request merged**